wpeverest/user-registration
# UR-169 Enhance - Size limit on profile picture field in edit profile page

### All Submissions:


### Changes proposed in this Pull Request:
Previously, there was no option to set the maximum file size in the profile picture field which could be verified at the time of profile picture upload, and don't allow profile pictures with a size greater than the max limit. This PR provides this option.

### How to test the changes in this Pull Request:

1. Navigate to the form builder and drag and drop the Profile Picture
2. In the Advance setting of field options of the profile picture field there will be an option to enter max file upload size.
3. Check in the edit profile and verify whether a picture with a size greater than the max limit is allowed or not.

### Types of changes:


### Other information:


### Changelog entry

> Enhance - Size limit on profile picture field in edit profile page.
